components/TextFields/tests/unit/OutlinedTextFieldColorThemerTests.m

```diff
@@ -154,4 +155,39 @@ - (void)_disabled_testThemerWithBaselineColorSchemeAppliesToMultilineTextFieldDi
   XCTAssertEqualObjects(textField.underline.disabledColor, controller.disabledColor);
 }
 
+- (void)testOutlinedTextFieldBorderStrokeColorTheming {
+  // Given
+  MDCTextField *textField = [[MDCTextField alloc] init];
+  MDCTextInputControllerOutlined *controller =
+      [[MDCTextInputControllerOutlined alloc] initWithTextInput:textField];
+  MDCSemanticColorScheme *colorScheme =
+      [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
+
+  // When
+  controller.borderStrokeColor = UIColor.redColor;
+  [MDCOutlinedTextFieldColorThemer applySemanticColorScheme:colorScheme
```

Comment:
Calling the themer is not relevant and is confusing for this test. let us remove it